docs/source/examples.daisy_chain_gap.ipynb

```diff
@@ -0,0 +1,746 @@
+    "\n",
+    "`(Initalizer, [args], {kwargs})`\n",
+    "\n",
+    "e.g. xtb would normally be called with \n",
+    "\n",
+    "`xtb_calc = XTB(method=\"GFN2-xTB\")`\n",
+    "\n",
+    "but instead in wfl scripts we define it as"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xtb_calc = (XTB, [], {\"method\": \"GFN2-xTB\"})"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare isolated atoms \n",
+    "\n",
+    "GAP requires reference (`e0`) energies for fitting. We construct `Atoms` objects with a single atom, evaluate them with the reference GFN2-xTB method and store in a file to later combine them with the training set. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "isolated_at_fname='isolated_atoms.xtb.xyz'\n",
+    "isolated_at = [Atoms(element, positions=[(0, 0, 0)]) for element in [\"H\", \"C\"]]\n",
+    "inputs = ConfigSet(isolated_at)\n",
+    "outputs = OutputSpec()\n",
+    "\n",
+    "inputs = generic.run(\n",
+    "    inputs=inputs,\n",
+    "    outputs=outputs,\n",
+    "    calculator=xtb_calc,\n",
+    "    properties=[\"energy\"],\n",
+    "    output_prefix=\"xtb_\"\n",
+    ")\n",
+    "\n",
+    "outputs = OutputSpec(isolated_at_fname)\n",
+    "for at in inputs:\n",
+    "    # GAP needs a cell for fitting, even for non-periodic structures.\n",
+    "    at.cell = [50,50,50]\n",
+    "    # set config_type for `gap_fit` to recognise the appropriate frames.  \n",
+    "    at.info[\"config_type\"] = \"isolated_atom\"\n",
+    "    outputs.store(at)\n",
+    "outputs.close()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generate initial structures\n",
+    "\n",
+    "We build this example on a small number of hydrocarbon molecules. Their connectivity is represented as SMILES strings and use RDKit to them into reasonable 3D geometries to start the molecular dynamics simulation with. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "all_smiles = [\n",
+    "    'CC1=CCC=CC(C)=C1C(C)C', \n",
+    "    'CC1(c2ccc(CC3CC=CC3)cc2)CC1', \n",
+    "    'C#CC[C@@H](CCC=C(C)C)C1CC1', \n",
+    "    'Cc1ccccc1CCCC1=CCCCC1', \n",
+    "    'C=CC1=CC[C@@H]2C[C@H]1C2(C)C', \n",
+    "    'C1=CCC(Cc2ccc(CC3CC3)cc2)C1', \n",
+    "    'C1=CC(c2ccccc2)=CCC1', \n",
+    "    'C/C=C/CCCC[C@H](C)C(C)(C)C', \n",
+    "    'C=C[C@@H]1C/C=C/CCCCCCCC1', \n",
+    "    'C[C@H](CC(C)(C)C)[C@@H](C)C(C)(C)C', \n",
+    "    'CC/C=C\\\\C[C@@H](C)c1cccc(C)c1C', \n",
+    "    'C=C1CC2c3ccccc3C1c1ccccc12']\n",
+    "\n",
+    "outputs = OutputSpec(\"1.ch.rdkit.xyz\")\n",
+    "inputs = smiles.run(all_smiles, outputs=outputs)"
```

Comment:
I would encourage less generic names than always using `inputs` and `outputs`.  Especially when you get to these more complex parts of the workflow, this looks weird, and in the next code block, where you're relying on `inputs` to contain the smiles _output_, it's extra weird.  How about something like `smiles_configs` ?